src/languages/en.ts

```diff
@@ -3298,11 +3298,21 @@ export default {
             reportFields: {
                 title: 'Report fields',
                 description: `Report fields let you specify header-level details, distinct from tags that pertain to expenses on individual line items. These details can encompass specific project names, business trip information, locations, and more.`,
-                pricing: {
-                    onlyAvailableOnPlan: 'Report fields are only available on the Control plan, starting at ',
-                    amount: '$9 ',
-                    perActiveMember: 'per active member per month.',
-                },
+                onlyAvailableOnPlan: 'Report fields are only available on the Control plan, starting at ',
+            },
+            netSuite: {
+                title: 'NetSuite',
+                description: `Enjoy automated syncing and reduce manual entries with the Expensify + NetSuite integration. Gain in-depth, realtime financial insights with native and custom segment support, including project and customer mapping.`,
```

Comment:
NAB: This copy reads a bit weird and is not consistent with Intacct below cc @garrettmknight @jamesdeanexpensify

Reply:
I'm happy to make updates, but they look *fairly* close. They're just a bit different because they support different fields/coding (or whatever you call them). Do you have any improvements to suggest?

Reply:
Their coding is different so their descriptions are going to be different. I do see one small unintentional difference in "real-time" vs. "realtime" @allroundexperts can you update "realtime" in the NetSuite blurb to be "real-time"?

Reply:
Ah @garrettmknight "realtime" is actually in our style - we've just kinda used it forever, so we've stuck with it as an Expensify stylistic choice.